docs/myriad-remote-distribution-configuration.md

```diff
@@ -0,0 +1,47 @@
+#Myriad Remote Distribution
+
+The Myriad Scheduler can be configured to automatically download and run the hadoop yarn binaries. This
+means you won't have to install and configure hadoop yarn on each machine.  However, this comes at the cost of
+downloading the binaries each time a node manager is spawned.  One should consider this tradeoff carefully.  Note
```

Comment:
Only once per node once https://issues.apache.org/jira/browse/MESOS-336 is implemented. And assuming you're smart enough to have a proxy at your local site, you only dl it from the internet once, then each slave downloads it from the local intranet.